test/unit/util/locking/FileSystemResourceLocker.test.ts

```diff
@@ -109,13 +109,22 @@ describe('A FileSystemResourceLocker', (): void => {
     await expect(locker.acquire(identifier)).rejects.toThrow(InternalServerError);
   });
 
-  it('clears the files in de lock directory after calling finalize.', async(): Promise<void> => {
+  it('clears the files in de lock directory upon calling initialize.', async(): Promise<void> => {
     await locker.acquire(identifier);
     await expect(readdir(lockFolder)).resolves.toHaveLength(1);
-    await locker.finalize();
-    await expect(readdir(lockFolder)).rejects.toThrow();
+    await locker.initialize();
+    await expect(readdir(lockFolder)).resolves.toHaveLength(0);
   });
 
+  it('stops proper-lock from throwing errors onCompromise after finalize was called.',
+    async(): Promise<void> => {
+      const onCompromise =
+          locker['defaultLockOptions'].onCompromised!; // eslint-disable-line @typescript-eslint/dot-notation
```

Comment:
```suggestion
      // eslint-disable-next-line @typescript-eslint/dot-notation
      const onCompromise = locker['defaultLockOptions'].onCompromised!; 
```
Although I'm not sure why dot notation is not used?

Reply:
I did not use dot notation, because I needed a way for accessing the private property `defaultLockOptions`, but maybe there is a better way of doing this?

I will rewrite this section as part of the requested change for still having `defaultLockOptions` outside of the `FileSystemResourceLocker` class, but I will still require access to a private property, so tips are welcome!